client/src/components/ActionButton/types.ts

```diff
@@ -5,7 +5,8 @@ export interface ActionButtonProps {
   label: string;
   labelColor: string;
   loading: boolean;
-  onClick: (event: MouseEvent) => void;
+  type: "submit" | "reset" | "button";
+  // onClick: (event: MouseEvent) => void;
```

Comment:
we should have onClick prop

we can just add additional prop `type` instead of replacing the current one